frontend/src/pages/ActivityReport/Pages/topicsResources.js

```diff
@@ -47,33 +54,21 @@ const TopicsResources = ({
             id="resourcesUsed"
             name="resourcesUsed"
             type="text"
-            inputRef={register({ required: true })}
+            inputRef={register()}
           />
         </div>
         <div className="smart-hub--form-section">
           <Label htmlFor="otherResources">Upload any resources used that are not available through ECLKC</Label>
           <Controller
-            name="otherResources"
+            name="attachments"
             defaultValue={[]}
             control={control}
             render={({ onChange, value }) => (
-              <FileUploader files={value} onChange={onChange} reportId={reportId} id="otherResources" />
+              <FileUploader files={value} onChange={onChange} reportId={reportId} id="attachments" />
             )}
           />
         </div>
       </Fieldset>
-      <Fieldset legend="Attachments" className="smart-hub--report-legend smart-hub--form-section">
-        <div id="attachments" />
-        <Label htmlFor="attachments">Upload any resources used that are not available through ECLKC</Label>
-        <Controller
-          name="attachments"
-          defaultValue={[]}
-          control={control}
-          render={({ onChange, value }) => (
-            <FileUploader files={value} onChange={onChange} reportId={reportId} id="attachments" />
-          )}
-        />
-      </Fieldset>
     </>
```

Comment:
Why did you remove the attachments? According to https://adhoc.slack.com/archives/C016NN6T0Q7/p1611949019010800?thread_ts=1611942489.010400&cid=C016NN6T0Q7 it should stay in.

Reply:
Oops! I'll add it back in!